estimagic/cli.py

```diff
@@ -31,14 +31,34 @@ def cli():
     is_flag=True,
     help="Don't open the dashboard in a browser after startup.",
 )
+@click.option(
+    "--jump",
+    is_flag=True,
+    help="Jump directly to the last iterations as specified in the rollover "
+    + "instead of replaying the optimization from the beginning.",
+)
 @click.option(
     "--rollover",
     default=10_000,
     help="After how many iterations convergence plots get truncated from the left.",
     type=int,
     show_default=True,
 )
-def dashboard(database, port, no_browser, rollover):
+@click.option(
+    "--frequency",
+    default=1,
+    help="Number of seconds to wait between checking for new entries in the database.",
+    type=float,
+    show_default=True,
+)
+@click.option(
+    "--update_chunk",
```

Comment:
Since `--no-browser` uses a dash instead of an underscore, we should do the same for `--update-chunk`

Reply:
Adressed in 9f3c5cbb9b9c3fe40548ed30da551737dfee5523